src/types/Patterns.ts

```diff
@@ -75,14 +75,16 @@ export const Patterns = {
   offerCard: {
     serviceName: /^[^-\s()'"#@.&](?!.*[%&?,';:!\s-]{2}).{2,200}$/,
     serviceType: /^([A-Za-z])$/,
-    shortDescriptionEN: /^[^-\s()'"#@.&](?!.*[%&?,';:!\s-]{2}).{10,120}$/,
-    shortDescriptionDE: /^[^-\s()'"#@.&](?!.*[%&?,';:!\s-]{2}).{10,120}$/,
+    shortDescriptionEN:
+      /^[^ @=<>*\-+#?%&_:;]([a-zA-ZÀ-ÿ0-9 !?@&#'"()[\]_\-+=<>/*.,;:%\r\n]){10,120}$/,
```

Comment:
The patterns for `shortDescriptionEN` and `shortDescriptionDE` are identical. Consider consolidating them into a single pattern to reduce duplication.
ex:
```
const descriptionPattern = /^[^ @=<>*\-+#?%&_:;]([a-zA-ZÀ-ÿ0-9 !?@&#'"()[\]_\-+=<>/*.,;:%\r\n]){10,120}$/

const shortDescriptionEN = descriptionPattern;
const shortDescriptionDE = descriptionPattern;
```

- A single `descriptionPattern` is defined to capture the common structure for description fields.
- The patterns for `shortDescriptionEN` and `shortDescriptionDE` are set to descriptionPattern.
- Adding comments explaining each pattern's purpose and the expected format of the description fields can enhance code understanding.
- Same can be done for `longDescriptionEN `

Reply:
@shubhamv-ss  thank you for your review and change request. I have now added the necessary changes.

Reply:
As per discussion with @ybidois, we have decided it makes most sense to remove all validation to allow all characters so we don't possiblly block a company onboarding a service because of a single special character.